src/components/List/List.tsx

```diff
@@ -0,0 +1,86 @@
+import classNames from 'classnames';
+import { Link } from 'react-router-dom';
+import { Person } from '../../types';
+import { PersonLink } from '../PersonLink';
+
+type Props = {
+  people: Person[] | null;
+  selectedPerson: string;
+};
+
+export const List: React.FC<Props> = (
+  {
+    people,
+    selectedPerson,
+  },
+) => {
+  if (!people) {
+    return null;
+  }
+
+  return (
+    <>
+      {people.length === 0
+        ? <p data-cy="noPeopleMessage">There are no people on the server</p>
+        : (
+          <table
+            data-cy="peopleTable"
+            className="table is-striped is-hoverable is-narrow is-fullwidth"
+          >
```

Comment:
Probably if people are null we shouldn't render this component or can render first condition.
```suggestion
  return (
    <>
      {people.length === 0 || !people
        ? <p data-cy="noPeopleMessage">There are no people on the server</p>
        : (
          <table
            data-cy="peopleTable"
            className="table is-striped is-hoverable is-narrow is-fullwidth"
          >
```